src/pill-input/pill.component.ts

```diff
@@ -13,19 +14,30 @@ import {
  * @class Pill
  */
 @Component({
-	selector: "ibm-pill",
+	selector: "ibm-pill, ibm-tag",
 	template: `
 		<span><ng-content></ng-content></span>
-		<button class="pill_close" (click)="doRemove($event)" type="button">
+		<button *ngIf="showCloseIcon" class="pill_close" (click)="doRemove($event)" type="button">
 			<ibm-static-icon icon="x" size="sm" classList="close_icon"></ibm-static-icon>
```

Comment:
@zvonimirfras ibm-static-icon uses the peretz icons by default. Are there plans to update it to use carbon icons?

Reply:
Yes, @cal-smith is working on the alternative.
In the mean time, replacing `svg`s in static-icon.component would do the trick. Cal can say is it worth doing at this time or not.